service/matching/db_task_manager.go

```diff
@@ -41,6 +42,9 @@ import (
 )
 
 const (
+	dbTaskInitialRangeID     = 1
+	dbTaskStickyTaskQueueTTL = 24 * time.Hour
```

Comment:
these 2 definitions are copy pasted from existing impl https://github.com/temporalio/temporal/blob/20bf977/service/matching/db.go#L43

so later db.go, taskReader.go, taskWriter.go, ackManager.go can simply be deleted

Reply:
once integration with new db task * are done